test/integration/fixtures/parallel/testworkerid1.mjs

```diff
@@ -0,0 +1,7 @@
+import {describe, it} from '../../../../index.js';
```

Comment:
`describe`, `it` , etc. are on the `global` object, so you can remove these imports.